Lists/Friends.lua

```diff
@@ -662,7 +663,7 @@ local function TooltipHook(self)
     local id = addon.UnitID(unit)
     if id then
     	GameTooltip:AddLine(' ')
-        GameTooltip:AddLine('Current Keystone')
+        GameTooltip:AddLine(L['CURRENT_KEY'])
```

Comment:
L['Current Keystone']

This goes in the tooltip and will look weird all capitalized.

Reply:
correction